### PR TITLE
Adding support for PipedProcess

### PR DESCRIPTION
**Pull request by vastradparayya**
Signed-off-by: Parayya Vastrad <parayya.vastrad@seagate.com>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: py-utils/src/utils/process.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
 import subprocess
+from subprocess import PIPE, Popen
```
Comment (sachinpunadikar) [inline py-utils/src/utils/process.py:19]: move this import statement to PipedProcess class  - do lazy import

**Review by rahul-tripathi-git — APPROVED** [review]
+1

**Review by sachinpunadikar — APPROVED** [review]
+1

**Pull request merged**